ui/settings/mainContent.css

```diff
@@ -25,7 +25,7 @@ QMessageBox
 
 QCheckBox
 {
-    background: white;
+    background: #FCFCFC;
```

Comment:
Actually by removing the whole rule, Qt will do the right thing (basically every widget has a white bg color in this rule set). Customizing style has to be done carfully and should not be done with a "sledge hammer", but more like with a "dissecting knife". :)